package.json

```diff
@@ -113,7 +113,6 @@
     "ajv-i18n": "^3.5.0",
     "ajv-merge-patch": "^4.1.0",
     "ajv-pack": "^0.3.1",
-    "autocannon": "^6.0.0",
```

Comment:
can you please add this back? This was used also by the benchmark scripts.

Reply:
Sorry i only looked in tests. Will add.